src/Model.js

```diff
@@ -184,6 +183,59 @@ const Model = class Model {
     }
 
     /**
+     * Update many-many relations for model.
+     * @param relations
+     */
+    _refreshMany2Many(relations) {
+        const ThisModel = this.getClass();
+        const This = this;
```

Comment:
I tried this out locally and I don't get any errors using this. I'm not sure what you mean by static property?

Man we really need to get the ci running! @tommikaikkonen